### PR TITLE
feat:add user + order models

### LINKED ISSUE #1: Create User + Order models

### PR DESCRIPTION
**Pull request by MarcosRS**
closes #1

### CODE REVIEW

File: db/models/order.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const {STRING, ENUM, ARRAY, JSON} = require('sequelize')
+
+module.exports = db => db.define('orders', {
```
Comment (pimpila) [inline db/models/order.js:5]: can you make 'orders' singular in the db.define arg?

File: db/models/order.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  status:{
+  	type: ENUM('completed', 'pending','cancelled','created'),
+  	defaultValue: 'created' 
+  },
+  glasses: {
+  	type:ARRAY(JSON),
+  	allowNull: falsenode
```
Comment (tashadesai) [inline db/models/order.js]: What is 'falsenode'?

File: db/models/order.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+}, { 	
+	getterMethods:{
+		totalPrice :function(){
```
Comment (pimpila) [inline db/models/order.js]: can you fix spacing in totalPrice: function

File: db/models/order.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const {STRING, ENUM, ARRAY, JSON} = require('sequelize')
+
+module.exports = db => db.define('orders', {
+  status:{
+  	type: ENUM('completed', 'pending','cancelled','created'),
+  	defaultValue: 'created' 
+  },
+  glasses: {
```
Comment (pimpila) [inline db/models/order.js:10]: we're wondering if this should be a virtual? wanna discuss all together?

**Review by pimpila — COMMENTED** [review]
check out single comments

**Review by tashadesai — COMMENTED** [review]
Left a few comments

**Pull request merged**